.github/workflows/run-shared-tests.yml

```diff
@@ -170,7 +170,7 @@ jobs:
 
       - name: Run tests
         # working-directory: funding-service-design-e2e-checks
-        run: uv run --frozen pytest --e2e-env ${{ inputs.env_name }} --tracing=retain-on-failure --browser chromium
+        run: cd tests && uv run --frozen pytest --e2e-env ${{ inputs.env_name }} --tracing=retain-on-failure --browser chromium
```

Comment:
I guess it's a bit strange the `working-directory` is commented out and then the command changes the directory in order to work? I'm not sure what the workaround here is doing but maybe it could be most clearly described by setting the `working-directory` attribute above

Reply:
(that sentence to really test the readers understanding of the word "work")

Reply:
@sfount I was wondering about this as well, I think the working directory is set by the parent here: https://github.com/communitiesuk/funding-service-design-workflows/blob/f71f03f5e10e98041bc10d73c49df8acd40d338f/.github/workflows/run-shared-tests.yml#L138 but we could try overwriting it here as an alternative.